src/test/java/api/support/fixtures/LoansFixture.java

```diff
@@ -324,6 +324,12 @@ public CheckInByBarcodeResponse checkInByBarcode(
       .at(servicePointId));
   }
 
+  public CheckInByBarcodeResponse checkInByBarcode(
+    IndividualResource item, UUID servicePointId) {
+
+    return checkInByBarcode(item, DateTime.now(DateTimeZone.UTC), servicePointId);
```

Comment:
Should this be DateTime.now or the current clock time?

Reply:
We can use the `checkInByBarcode` method which takes 3 arguments if we need some tricks with check-in date. 

This is overloaded method which uses current date time for check-in. I'd leave it in current format.

Reply:
Ok, I'll resolve this comment